packages/rest/src/lib/utils/utils.ts

```diff
@@ -167,3 +167,12 @@ export async function onRateLimit(manager: RequestManager, rateLimitData: RateLi
 		throw new RateLimitError(rateLimitData);
 	}
 }
+
+/**
+ * Calculates the default avatar type for a given user id.
+ *
+ * @param userId - The user id to calculate the default avatar type for
+ */
+export function calculateUserDefaultAvatarType(userId: string) {
+	return Number(BigInt(userId) >> 22n) % 5;
```

Comment:
This needs to be adjusted to `% 6` now according to api-docs, new pink default avatar was added

Reply:
this is not yet reflected on the client though

Reply:
Anyway this is already documented